config/settings/base.py

```diff
@@ -40,8 +40,8 @@
 # DATABASES
 # ------------------------------------------------------------------------------
 # https://docs.djangoproject.com/en/dev/ref/settings/#databases
-DATABASES = {"default": env.db("DATABASE_URL")}
-DATABASES["default"]["ATOMIC_REQUESTS"] = True
+DATABASES = {"default": "sqlite:///db.sqlite3"}
+# DATABASES["default"]["ATOMIC_REQUESTS"] = True
```

Comment:
are these changes for local development?

Reply:
reverted